cmd/backup/main.go

```diff
@@ -93,17 +96,66 @@ type script struct {
 	webdavClient *gowebdav.Client
 	logger       *logrus.Logger
 	sender       *router.ServiceRouter
+	template     *template.Template
 	hooks        []hook
 	hookLevel    hookLevel
 
-	start  time.Time
-	file   string
-	output *bytes.Buffer
+	file  string
+	stats *Stats
 
-	c *config
+	c *Config
 }
 
-type config struct {
+// ContainersStats stats about the docker containers
+type ContainersStats struct {
+	All        int
+	ToStop     int
+	Stopped    int
+	StopErrors int
+}
+
+// BackupFileStats stats about the created backup file
+type BackupFileStats struct {
+	Name      string
+	FullPath  string
+	SizeBytes int64
+	SizeBin   string
```

Comment:
Considering that you already added a `FuncMap` for the template I wonder if these calculations should rather happen on-demand in the template?

Reply:
Completely agree!